cmake/CPackSetup.cmake

```diff
@@ -51,7 +51,7 @@ set(CPACK_FREEBSD_PACKAGE_ORIGIN "misc/libdpp")
 set(CPACK_RPM_PACKAGE_LICENSE "Apache 2.0")
 set(CPACK_PACKAGE_CONTACT "https://discord.gg/dpp") # D++ Development Discord
 set(CPACK_DEBIAN_PACKAGE_DEPENDS "libsodium23 (>= 1.0.17-1), libopus0 (>= 1.3-1)")
-set(CPACK_RPM_PACKAGE_REQUIRES "libsodium >= 1.0.17, opus >= 1.3.1")
+set(CPACK_RPM_PACKAGE_REQUIRES "libsodium >= 1.0.20-1, opus >= 1.3.1")
```

Comment:
doesnt ">= 1.0.17" already cover 10.0.20-1?

Reply:
doesn't seem like it, when building it says the version wasn't found &/or its below the requirements

Reply:
Yeah, this aligns with https://github.com/brainboxdotcc/DPP/issues/1142.

The current version of libsodium requested seems to just not work on RPM packages.